content/maxwell3_fdem/emi_tutorial/amplitudeandphase.rst

```diff
@@ -0,0 +1,129 @@
+.. _amplitudeandphase:
+
+Amplitude and phase
+===================
+
+.. figure:: ./images/Concepts_3loops_only_K.png
+   :align: center
+   :scale: 60%
+   :name: Concepts_3loops_only_K
+
+   Conceptual diagram for 3-loops system.
+
+Based upon the induced current in the body (Loop2), in :ref:`induced_currents_body`, here we consider amplitude and phase of the secondary magnetic field (:math:`H^s`) normalized by primary magnetic field (:math:`H^p`), :math:`\frac{H^s}{H^p}`, measured at Rx (Loop3). For this, we first consider the primary magnetic field at body (Loop2), :math:`H^p_2`, resulting from a current, :math:`i^p`, flowing in the Loop1, which can be expressed as
+
+.. math::
+	H^p_2(t) = K_{12} i^p(t),
```

Comment:
for notation, I think it is dangerous to use `i` - easily confused with the imaginary unit. I would suggest saying `I^p(\omega,t) = I^p cos(\omega t)` What do you think of that?